moveit_experimental/moveit_jog_arm/src/low_pass_filter.cpp

```diff
@@ -47,6 +48,18 @@ namespace moveit_jog_arm
 LowPassFilter::LowPassFilter(double low_pass_filter_coeff)
 {
   filter_coeff_ = low_pass_filter_coeff;
+  if (low_pass_filter_coeff == 1.0)
+  {
+    ROS_WARN_NAMED(LOGNAME, "LowPassFilter constructed as a 2 tap boxcar FIR");
+  }
+  else if (low_pass_filter_coeff == -1.0)
+  {
+    ROS_FATAL_NAMED(LOGNAME, "LowPassFilter coeff value of -1.0 will result in divide by zero error");
+  }
+  else if (low_pass_filter_coeff < 1.0)
+  {
+    ROS_ERROR_NAMED(LOGNAME, "LowPassFilter constructed as an unstable IIR");
```

Comment:
Would it be better if the warning was tiny bit more explicit like this:
`"LowPassFilter constructed as an unstable IIR (Infinite Impulse Response)."`?